src/twisted/newsfragments/10343.misc

```diff
@@ -0,0 +1 @@
+Enable Python 3.11 for CI and fix the remaining broken tests.
```

Comment:
This is the user-facing changelog entry, so it should read as a thing that users are interested in.  This should be a `.feature`, and it should be more like this:

```suggestion
Twisted now officially supports Python 3.11.
```

Reply:
(Users don't care about the configuration details of our CI or "broken tests"; they care about whether or not we stand behind using Twisted with a particular Python version.)

Reply:
+1 for having this as a feature and using the text suggested by Glyph.

Reply:
Updated & moved this to a feature.